context.go

```diff
@@ -0,0 +1,109 @@
+package turn
+
+import (
+	"crypto/md5" // #nosec
+	"net"
+	"strings"
+
+	"github.com/pion/stun"
+	"github.com/pkg/errors"
+)
+
+// Context is used to hold the context of a request
+type Context struct {
+	conn    net.PacketConn
+	srcAddr net.Addr
+	msg     *stun.Message
+
+	realm       string
+	authHandler AuthHandler
+
+	messageIntegrity stun.MessageIntegrity
+}
+
+// NewContext creates a new instance of Context.
+func NewContext(conn net.PacketConn, srcAddr net.Addr, m *stun.Message, realm string, authHandler AuthHandler) *Context {
```

Comment:
If possible I would try to push everything into `internal` possible. Just so if people jump into `pion/turn` and want to read it can make it as easy as possible.

Reply:
Can't agree and more, thanks.

Reply:
I add `server` and `client` sub package to `internal ` and move `context.go`  to `server`  package.